hera_mc/utils.py

```diff
@@ -27,3 +27,14 @@ def calculate_obsid(starttime):
         raise ValueError('starttime must be an astropy Time object')
 
     return int(floor(starttime.gps))
+
+
+def get_iterable(x):
+    """Helper function to ensure iterability."""
+    if isinstance(x, str):
+        return (x,)
+    else:
+        if isinstance(x, collections.Iterable):
```

Comment:
This will work for the core built-in collection types but probably not possible user-defined iterable types. If we care about that, the more to-the-point approach would be to do something like:

```
try:
    iter(x)
except TypeError:
    return (x,)
return x
```